index.js

```diff
@@ -48,6 +48,7 @@ class KeyringController extends EventEmitter {
 
     this.encryptor = opts.encryptor || encryptor;
     this.keyrings = [];
+    this.unsupportedKeyrings = [];
```

Comment:
Nit: We should probably prefix this with an underscore, to denote that it's not meant to be accessed directly.

I had meant to hide the other properties as well, during the TypeScript migration. I guess this could wait until then too.

Reply:
Good point, I added underscore to it.